### PR TITLE
[Meru AB] Assistant description suggestions

### LINKED ISSUE #3885: [Meru AB] Suggestions for Naming screen - Descriptions
Details on UX for these suggestions on  **[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4900-39387&mode=design&t=CrHO9TDk6FhlGRRq-4)** (also, see  context & discussion in the [making issue](https://github.com/dust-tt/decisions/issues/164))
- Dust app 
  - taking as input instructions, action, names of existing assistants
  - output a possible description
  - brief, max 3 lines
  - probably use gpt-3.5-turbo for this
- Run on instructions/action change (3s delay for typing fields)
- Depends on #3785

### PR DESCRIPTION
**Pull request by philipperolet**
Fixes #3885

Notes (cc @Duncid)
---
- Limited to 2 suggestions
- Needed to allow 2 lines in the button for some suggestions
  - another option would be a single suggestion (1 line) and a refresh button (a bit longer to do, not that much)
- First version, might need tweaking

Screenshot from 2024-02-28 22-32-15

### CODE REVIEW

**Comment by spolu** [comment]
I think the UI is not great here. This should be a button that will fill the description field instead of a suggestion no?

**Comment by philipperolet** [comment]
Those are all buttons that fill the description field if clicked here
Do you think they should look different?

File: front/components/assistant_builder/NamingScreen.tsx [hunk 1]
```diff
@@ -264,6 +295,34 @@ export default function NamingScreen({
...
+                </div>
+                {descriptionSuggestions.suggestions
+                  .slice(0, 2)
+                  .map((suggestion, index) => (
+                    <Button
+                      label={suggestion}
+                      variant="secondary"
+                      key={`description-suggestion-${index}`}
+                      size="xs"
+                      className="!h-auto min-h-7 max-w-[45%] !whitespace-normal"
```
Comment (flvndvd) [inline front/components/assistant_builder/NamingScreen.tsx:313]: 😱 ❗

**Review by flvndvd — COMMENTED** [review]
Not a big fan of the design 🤔. IMHO, using a button to display the suggestions clutters a lot the UI.

**Comment by Duncid** [comment]
Indeed buttons for suggestions not working very well.
Can you push it and Dust feature flag it so I can check it out "live" a little before iterating?

**Comment by philipperolet** [comment]
Sorry ed just seeing your com now. After I felt unsatisfaction from button with long text, + both @spolu & @flavien pushed back on the UI in screenshot, I moved to a sparkle iconbutton to generate a description as they suggested => pushed it on front-edge so you can test this version (with sparkle button), and at your request I can push the other version (with long test in buttons), so you can test both
image

**Comment by philipperolet** [comment]
I'm going to go ahead and merge so people can easily try the sparkle button version; and I'll deploy the big descriptions button on front edge if you want me to

**Pull request merged**